### ISSUE TITLE
Dragging 

### ISSUE THREAD

**Issue by jonahgreenthal** [react-dropzone/react-dropzone#755]
**Do you want to request a *feature* or report a *bug*?**
- [x] I found a bug
- [ ] I want to propose a feature

**What is the current behavior?**
Dragging an image from another browser window to a Dropzone doesn't work in Chrome.

**If the current behavior is a bug, please provide the steps to reproduce.**

1. Open https://react-dropzone.js.org and scroll to the "Previews" section
1. In another window of the same browser, open an image, e.g. https://www.google.com/images/branding/googlelogo/2x/googlelogo_color_272x92dp.png
1. Drag and drop the image into the Dropzone

**What is the expected behavior?**
Dropzone should process the image and show a preview.

This works in Safari 12.0.3 (14606.4.5), and possibly in other browsers. However, in Chrome 72.0.3626.96 (both Windows and Mac), nothing happens. I believe `isDragActive` never becomes true.

**Please mention other relevant information such as the browser version, Operating System and react-dropzone version.**

**Comment by rolandjitsu** [comment]
@jonahgreenthal that is expected behaviour.

You are not dragging a file, per say, you are dragging an html element (if you look at the data transfer item type, it's `text/html`), and we intentionally ignore those types because this lib only handles files from your system.

I think [react-dnd](https://github.com/react-dnd/react-dnd) might have what you need.

**Comment by jonahgreenthal** [comment]
So the fact that it works in Safari is just a complete coincidence/accident?

**Comment by rolandjitsu** [comment]
That's probably a a bug or Safari reports that as a file type. I will investigate this tomorrow.

**Comment by rolandjitsu** [comment]
That seems to be the case, Safari reports the drag item type as file. I'm working on figuring out what to do about this, but in principle, we'd like to only handle files with this lib and ignore other types (links, html elements, etc.).

**Comment by jonahgreenthal** [comment]
That's unfortunate; it was super useful to me when it worked. I'll look at react-dnd…but if you're willing to make this package support that functionality across browsers, I'd sure be grateful. Thanks!

**Comment by rolandjitsu** [comment]
@jonahgreenthal unfortunately, Safari seems to be the only browser that can get the file data when an image is dropped from a different browser window.

I have tried using [FileSystemFileEntry.file()](https://developer.mozilla.org/en-US/docs/Web/API/FileSystemFileEntry/file) which just throws in Chrome and Firefox. I also tried [DataTransferItem.getAsFile()](https://developer.mozilla.org/en-US/docs/Web/API/DataTransferItem/getAsFile) which returns `null`.

Bottom line, we will not be able to add support for this at the moment, unless we find some new APIs or tricks to achieve the same behaviour as Safari has in Chrome, Firefox and Edge.

It's possible that the other browser might follow suit and support this behaviour in the future, but I wouldn't keep my hopes up.

**Comment by jonahgreenthal** [comment]
Okay, thanks for the info. I was not able to get this working with react-dnd, though I admittedly didn't spend all that much time on it. If you happen to know specifically how to do it (e.g., you know a particular tutorial that covers file uploads with react-dnd) that would be great, but I understand that's not your job :-) Thanks for looking at this.

**Comment by rolandjitsu** [comment]
@jonahgreenthal I think that [examples/other/native-files](http://react-dnd.github.io/react-dnd/examples/other/native-files) should cover drag 'n' drop for files. Though, I doubt that it will behave any different for your use case (besides Safari). Let me know if react-dnd will work for you.

**Comment by jonahgreenthal** [comment]
Thanks. That's just like Dropzone: it works in Safari (to drag and drop an image from another browser window), but not Chrome.

**Comment by rolandjitsu** [comment]
@jonahgreenthal thanks for letting me know. Let's hope that APIs will evolve in the future (or get fixed) and allow for such behavior across browsers.
